inc/views/template_parts.php

```diff
@@ -117,10 +117,7 @@ public function render_featured_post() {
 			return;
 		}
 
-		$wrapper_classes = [];
-		if ( ! neve_is_new_skin() ) {
-			$wrapper_classes[] = 'row';
-		}
+		$wrapper_classes  = [];
 		$wrapper_classes  = apply_filters( 'neve_posts_wrapper_class', $wrapper_classes );
```

Comment:
not necessarily, we can remove the first `$wrapper_classes` and pass the default value to `apply_filters` call as hard-coded